packages/ui/src/layout/wui-flex/index.ts

```diff
@@ -0,0 +1,68 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { styleMap } from 'lit/directives/style-map.js'
+import styles from './styles'
+import type { Color, BorderRadius, Spacing } from '../../utils/TypesUtil'
+
+@customElement('wui-flex')
+export class WuiFlex extends LitElement {
+  public static styles = [styles]
+
+  @property() public backgroundColor?: Color
+
+  @property() public borderRadius?: BorderRadius
+
+  @property() public flexDirection?: string
```

Comment:
Instead of these being just strings, any chance we can hook into actual style types for each, to get correct auto-complete? If not, can we define them in type util and use here.

I saw you did something like this in preset util, so maybe types can be derived from there?